server/api/utils/email.ts

```diff
@@ -20,13 +20,38 @@ const getItemAgeDescription = (age: number): string => {
 };
 
 const transporter = nodemailer.createTransport({
-    service: "gmail",
+    host: "smtp.office365.com",
+    port: 587,
     auth: {
-        user: "blueprintpregnancycentre@gmail.com", // TODO: Change later once TPC gives us an email address
+        user: "noreply@pregnancycentre.ca",
         pass: process.env.EMAIL_PASSWORD
     }
 });
 
+async function sendRejectionEmail(firstName: string, lastName: string, email: string, item: Item) {
+    let htmlString = `
+        <body><p>Dear ${firstName} ${lastName}, <p>
+        <p>Thank you for connecting with the Pregnancy Centre. After reviewing your donation form we are not able to receive the following donation at this time: </p>
+        <p>Donated item: ${item.name}<br>
+        Condition of Item: ${item.condition}<br>
+        Age of item: ${getItemAgeDescription(item.age)}<br>
+        Quantity: ${item.quantity}<br>
+    `;
+    if (item.description != null) {
+        htmlString += `Item Description: ${item.description}`;
+    }
+    htmlString +=
+        "</p><p>Please contact us with any questions at info@thepregnancycentre.ca and please consider donating other items in the future.</p>The Pregnancy Centre</body>";
+
+    await transporter.sendMail({
+        from: '"no-reply " <noreply@pregnancycentre.ca>', // sender address
+        to: email, // list of receivers
+        subject: "TPC's Donation Review", // Subject line
+        text: "Your donation form has been approved", // plain text body
```

Comment:
i think this should say "Your donation form has been rejected"

Reply:
Fixed.